.ci_support/migrations/hdf51141.yaml

```diff
@@ -0,0 +1,7 @@
+__migrator:
+  build_number: 1
+  kind: version
+  migration_number: 1
+hdf5:
+- 1.14.1
+migrator_ts: 1687165815.3104627
```

Comment:
Have added the new `hdf5` migrator based on this commit ( https://github.com/conda-forge/conda-forge-pinning-feedstock/commit/77584f273c3fac007af0d55cd22b137ba7d02de9 )